File: Log Formula/log.cs
class Program
{
    public static void Main(string[] args)
    {
        var arr = new ArrayList();  

        for(int y = 1; y < 10; y++)
        {
            var x = Math.pow(3, y); 
            arr.add(x); 
        }

        Console.WriteLine(arr); 
    }
}

File: Log Formula/log.js
let arr = []; 

//change the 10 to how big you want the array to be
for(let y = 1; y < 10; y++) {

    //change the 3 here to whatever you want to log by
    let x = Math.pow(3, y); 
    arr.push([x, y]); 
}

console.table(arr); 